rpcs3/Emu/Cell/Modules/sys_lwcond_.cpp

```diff
@@ -250,13 +250,14 @@ s32 sys_lwcond_wait(ppu_thread& ppu, vm::ptr<sys_lwcond_t> lwcond, u64 timeout)
 		}
 
 		// restore owner and recursive value
-		const auto old = lwmutex->vars.owner.exchange(tid);
+		lwmutex->vars.owner.store(tid);
 		lwmutex->recursive_count = recursive_value;
 
+		/* Commented out as this check doesnt exist in liblv2, and sometimes it doesn't pass (Just Cause 2)
```

Comment:
I agree it means something is wrong, but the game can, at any point, edit the mutex himself and mess this up - they are in user space. If it does so, we're not supposed to be the ones stopping it, are we?